### DISCUSSION TITLE
Lift z axis before moving?

### DISCUSSION THREAD

**Discussion by pingu6666** [Jyers/Marlin#485] [Q&A] [answered]
How can i make the printer lift the z axis slightly before doing anything else at the start of the print ? 
my printhead tends to drop by itself, so it can end up resting on the bed, and guages with this firmware as it moves the x and y straight right away, and will scratch the bed :(

really happy with the firmware otherwise :)

**Comment by Cal-LF** [comment]
I had to induce some drag to keep the X carriage from dropping. The brass Z nut can be offset a little.

**Comment by Jyers** [comment, marked as answer]
You can also add a relative z lift to your start gcode before the printer does anything else.

**Comment by G4GNX** [comment]
You could fit an anti-backlash nut, which should prevent the lead screw from turning.
You also might want to check the eccentric nuts on the gantry. It sounds as if they're too loose, so there's no friction on the rollers.
